examples/triton/tensorflow/build_bento.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+import os
+import typing as t
+import argparse
+
+import attr
+import torch
+
+import bentoml
+from bentoml import Bento
+from bentoml._internal.utils import resolve_user_filepath
+from bentoml._internal.bento.build_config import BentoBuildConfig
+from bentoml._internal.configuration.containers import BentoMLContainer
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--tag", type=str, default=None)
+    parser.add_argument("--gpu", action="store_true", default=False)
+    parser.add_argument("--override", action="store_true", default=False)
+
+    args = parser.parse_args()
+
+    tag = "triton-integration-tensorflow"
+    if args.tag:
+        tag = f"triton-integration-tensorflow:{args.tag}"
+
+    try:
+        bentos = bentoml.get(tag)
+        if args.override:
+            bentoml.delete(tag)
+            raise bentoml.exceptions.NotFound("'--override', rebuilding bentos.")
```

Comment:
What's the reason we want an override switch?

Reply:
Oh I think I should remove that. I was building a lot of models during testing so I don't want to pollute and repeat a lot of these bentos